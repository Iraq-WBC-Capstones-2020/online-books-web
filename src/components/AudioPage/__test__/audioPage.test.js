import React from 'react';
import renderer from 'react-test-renderer';
import AudioPageSection from './../audioPage';

test('audioPage snapshot', () => {
  const tree = renderer.create(<AudioPageSection />).toJSON();
  expect(tree).toMatchSnapshot();
});
