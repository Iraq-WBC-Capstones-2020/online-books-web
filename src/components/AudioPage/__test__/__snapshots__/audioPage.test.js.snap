// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`audioPage snapshot 1`] = `
   <div className="container mt-5" data-testid="container">
      <div className="row" data-testid="row">
        <div className="col-12 col-sm-5" data-testid="col">
          <div className="mt-5 d-flex align-items-center justify-content-center">
            <img
              src={bookInfo.bookCover}
              width="353"
              height="498"
              alt="bookCover"
            />
          </div>
        </div>
        <div className="col-12 col-sm-7" data-testid="col">
          <div className="mt-5">
            <h1 className="blue-text">{bookInfo.bookName}</h1>
            <Paragraph>{bookInfo.description}</Paragraph>
          </div>
          <div className="mt-5">
            <AudioPlayerPage />
          </div>
        </div>
      </div>
    </div>
`;
